docs-ref-conceptual/manage-azure-subscriptions-azure-cli.md

```diff
@@ -1,58 +1,161 @@
+* To learn how to sign in to a specific tenant, see [Sign in with the Azure CLI](./authenticate-azure-cli.md).
 
 ## Change the active subscription
 
-To access the resources for a subscription, switch your active subscription or use the `--subscription` argument. Switching your subscription
-for all commands is done with [az account set](/cli/azure/account#az-account-set).
+To access the resources for a subscription, switch your active subscription or use the **--subscription** argument. Switching your subscription for all commands is done with [az account set](/cli/azure/account#az-account-set).
 
 To switch your active subscription:
 
 1. Get a list of your subscriptions with the [az account list](/cli/azure/account#az-account-list) command:
 
-    ```azurecli-interactive
+    ```azurecli
     az account list --output table
     ```
-2. Use `az account set` with the subscription ID or name you want to switch to.
 
-    ```azurecli-interactive
+   This command lists all the subscriptions you can access. If you don't see a subscription you expect, add the **--refresh** parameter to get the most current list of subscriptions.
+
+   > [!TIP]
+   > The **--output** parameter is a global parameter, available for all commands. The `table` value presents output in a friendly format. For more information, see [Output formats for Azure CLI commands](/cli/azure/format-output-azure-cli).
+
+1. To see the subscription you're currently using, run the [az account show](/cli/azure/account#az_account_show) command:
+
+   ```azurecli
+   az account show --output table
+   ```
+
+1. To switch to a different subscription, use [az account set](/cli/azure/account#az-account-set) with the subscription ID or name you want to switch to.
+
+    ```azurecli
     az account set --subscription "My Demos"
     ```
 
-To run only a single command with a different subscription, use the `--subscription` argument. This argument takes either a subscription ID or subscription name:
+   Your subscriptions have both a name and an ID, which is a GUID. You can use either for these commands. If you use a name that includes spaces, use quotation marks.
 
-```azurecli-interactive
+To run only a single command with a different subscription, use the **--subscription** argument:
+
+```azurecli
 az vm create --subscription "My Demos" --resource-group MyGroup --name NewVM --image Ubuntu
-```
\ No newline at end of file
+```
+
+## Azure CLI config file
+
+There's a configuration file, named *config,* in this location on the computer where you run Azure CLI:
+
+- *$HOME/.azure* on Linux and macOS
+- *%USERPROFILE%\.azure* on Windows
+
+In general, don't edit this file directly. Instead, use the [az config](/cli/azure/config) commands.
+
+Try setting your default location by using the [az config set](/cli/azure/config#az_config_set) command:
+
+```azurecli
+az config set defaults.location=eastus
```

Comment:
Please remove any example that is something other than for accounts / subscriptions.  i.e. this article is not about location or even az config, so let's stay focused on working with az config for subscriptions.